src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/SqlCustomTagStore.cs

```diff
@@ -87,9 +87,34 @@ public Task<IEnumerable<CustomTagEntry>> GetAllCustomTagsAsync(CancellationToken
             throw new System.NotImplementedException();
         }
 
-        public Task DeleteCustomTagAsync(string tagPath, string vr, CancellationToken cancellationToken = default)
+        public async Task DeleteCustomTagAsync(string tagPath, string vr, CancellationToken cancellationToken = default)
         {
-            throw new System.NotImplementedException();
+            if (_schemaInformation.Current < SchemaVersionConstants.SupportCustomTagSchemaVersion)
+            {
+                throw new BadRequestException(DicomSqlServerResource.SchemaVersionNeedsToBeUpgraded);
```

Comment:
Feature not supported right? notsupportedexception?

Reply:
Following FHIR